integrations/marketo-v2/lib/index.js

```diff
@@ -141,6 +177,16 @@ Marketo.prototype.page = function(page) {
   });
 };
 
+Marketo.prototype.setupAndSubmitForm = function(traits, form) {
+  console.log('PRIME')
```

Comment:
Do we need to keep this console.log ?

Reply:
No no, this was for testing purposes. My bad!